main.cpp

```diff
@@ -92,6 +92,76 @@ good to go!
  */
 
 #include <iostream>
+
+struct FloatType
+{
+    float add( float lhs, float rhs )
+    {
+        return lhs + rhs;
+    }
+    float subtract( float lhs, float rhs )
+    {
+        return lhs - rhs;
+    }
+    float multiply( float lhs, float rhs )
+    {
+        return lhs * rhs;
+    }
+    float divide( float lhs, float rhs )
+    {   
+        if (rhs == 0.0f) std::cout << "\nwarning, floating point division by zero returns 'inf' !" << std::endl;
```

Comment:
put `if()` bodies on their own line to improve the readability of the code
- [ ] revise